docs/template_plugin/tests/functional/shared_tests_instances/single_layer_tests/split.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+
+#include <vector>
+
+#include "single_layer_tests/split.hpp"
+#include "common_test_utils/test_constants.hpp"
+
+using namespace LayerTestsDefinitions;
+
+namespace {
+
+const std::vector<InferenceEngine::Precision> netPrecisions = {
+        InferenceEngine::Precision::FP32
+};
+
+INSTANTIATE_TEST_CASE_P(NumSplitsCheck, SplitLayerTest,
+                        ::testing::Combine(
+                                ::testing::Values(1, 2, 3, 5, 6, 10, 30),
+                                ::testing::Values(0, 1, 2, 3),
+                                ::testing::ValuesIn(netPrecisions),
```

Comment:
Minor comment, could you use one style in parameters? I am talking about the vector `netPrecisions`

Reply:
What should be changed. Could you explain, please?

Reply:
I mean `::testing::ValuesIn(InferenceEngine::Precision::FP32),`

Reply:
Done